docs: revise Flex manual text on pipette tip sterility

jwwojak opened the pull request:
# Overview

Changes to the sterility certification that ships with our pipette tips could help improve related text in the Flex manual. This is in Labware > Labware types > Tips and tip racks.

The proposed changes here do some small reorganization and improve clarity about our sterility specs based on the revised sterility cert (linked in jira below).

Sandbox: https://sandbox.docs.opentrons.com/docs-pipette-tip-sterility/

RTC-868

## Test Plan and Hands on Testing

Read for accuracy, clarity.

## Changelog

Changes `types.md`, specifically revises the "Tips and tip racks" intro and adds H3s with more technical detail about sterility practices.

## Review requests

Do these changes seem reasonable and helpful?

## Risk assessment

Low, minor revisions to existing content.

jwwojak committed:
first draft changes

jwwojak committed:
revision 2

Break up a long sentence

jwwojak committed:
Small grammar fix

ecormany commented on docs/flex/docs/labware/types.md:
```diff
@@ -55,13 +55,17 @@ For a full example of a well plate lid, reference the [Opentrons Tough PCR Auto-
+
+### Tip sterility
+
+Sealed, unopened packages of Opentrons pipette tips are certified to be free of contaminants. Tips are sterilized based on ANSI/AAMI/ISO 11137 standards with irradiation at a Sterility Assurance Level (SAL) of 10<sup>-6</sup>. Each package includes a letter describing our test procedures and certifies that every lot is free of common contaminants such as DNase/RNase, endotoxins (Pyrogens), human  ...
```
lowercase `pyrogens`

emilyburghardt replied:
would also recommend `contaminants such as DNase, RNase, endotoxins`... instead of the `/` but that's a nitpick

jwwojak replied:
Making recommended changes.

ecormany reviewed (approved):
one tiny change, then good to go.

jwwojak committed:
Reviewer recommended changes

- remove / between DNase and RNase. Separate with comma.

- lc "Pyrogens" --> pyrogens
